docs/integrations/microsoft-azure/opentelemetry/sql-server-opentelemetry.md

```diff
@@ -31,9 +31,9 @@ Following are the [Fields](https://help.sumologic.com/docs/manage/fields/) which
 
 Make sure logging is turned on in SQL Server. Follow [this documentation](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/scm-services-configure-sql-server-error-logs?view=sql-server-ver15) to enable it.
 
-The Microsoft SQL Server App's queries and dashboards depend on logs from the SQL Server ERRORLOG, which is typically found in: `C:\Program Files\Microsoft SQL Server\MSSQL<version>.MSSQLSERVER\MSSQL\Log\ERRORLOG*`
+The Microsoft SQL Server App's queries and dashboards depend on logs from the SQL Server ERRORLOG, which is typically found in: `/var/opt/mssql/log/errorlog`.
```

Comment:
This is incorrect. Since this doc is talking about a app which supports windows.